README.md

```diff
@@ -2,36 +2,40 @@
+> [![EFIBuild](https://github.com/qubic/core/actions/workflows/efi-build-develop.yml/badge.svg?branch=main)](https://github.com/qubic/core/actions/workflows/efi-build-develop.yml)
 > <br>
 > DEVELOP (current version we're working on) <br>
-> [![EFIBuild](https://github.com/qubic-network/core/actions/workflows/efi-build-develop.yml/badge.svg?branch=develop)](https://github.com/qubic-network/core/actions/workflows/efi-build-develop.yml)
+> [![EFIBuild](https://github.com/qubic/core/actions/workflows/efi-build-develop.yml/badge.svg?branch=develop)](https://github.com/qubic/core/actions/workflows/efi-build-develop.yml)
 
 ## Prerequisites
-To run a qubic node, you need the following parts:
-- Bare Metal Server/Computer with at least 8 Cores (high CPU frequency with AVX2 support)
-- At least 128GB memory
+To run a qubic node, you need the following spec:
+- Bare Metal Server/Computer with at least 8 Cores (high CPU frequency with AVX2 support) Check supported CPUs [here](https://www.epey.co.uk/cpu/e/YTozOntpOjUwOTc7YToxOntpOjA7czo2OiI0Mjg1NzUiO31pOjUwOTk7YToyOntpOjA7czoxOiI4IjtpOjE7czoyOiIzMiI7fWk6NTA4ODthOjY6e2k6MDtzOjY6IjQ1NjE1MCI7aToxO3M6NzoiMjM4Nzg2MSI7aToyO3M6NzoiMTkzOTE5OSI7aTozO3M6NzoiMTUwMjg4MyI7aTo0O3M6NzoiMjA2Nzk5MyI7aTo1O3M6NzoiMjE5OTc1OSI7fX1fYjowOw==/)
+- At least 128GB of storage
 - 1Gb/s synchronous internet connection
-- An USB Stick or SSD/HD attached to the Computer via USB
-- An UEFI Bios
+- A USB Stick or SSD/HD attached to the Computer via USB
+- An UEFI Bios 
 
-> You will need the current `spectrum, universe, and contract` files to be able to start qubic. The latest files can be found in our #computor-operator channel on the Qubic Discord server: https://discord.gg/qubic (ask there for the files).
+> You will need the current `spectrum, universe, and contract` files to be able to start Qubic. The latest files can be found in our #computor-operator channel on the Qubic Discord server: https://discord.gg/qubic (inquire there for the files).
 
 ### Prepare your USB Stick/SSD/HD
-1. Your Qubic Boot device should be formatted as FAT32 and have the label QUBIC.
+1. Your Qubic Boot device should be formatted as FAT32 with the label QUBIC.
 ```bash
 # sample command in linux
 mkfs.fat -F 32 -n QUBIC /dev/sda
 ```
 If you have a disk and want to use partitions, this is possible too. use `gdisk`.
 ```bash
 gdisk /dev/sda
+
 # remove all existing partition with d command
 # add the qubic partition with n command
-# it is recommended to use <1TB of partition size; let be the start sector. end sector can be specified with size. eg: 200G.
+# it is recommended to use <1TB of partition size; the start sector and the end sector can be specified with size. eg: 200G.
 # set the type of partition to ef00
+
+echo -e "o\nY\nd\nn\n\n\n+200G\n\nt\n\nef00\nw\nY" | gdisk /dev/sda
+

```

Comment:
I cannot verify this part. Okay @krypdkat @J0ET0M ?

Reply:
the command looks correct to me